esmtools/carbon.py

```diff
@@ -125,3 +128,129 @@ def potential_pco2(t_insitu, pco2_insitu):
     t_sfc = t_insitu.isel(depth=0)
     pco2_potential = pco2_insitu * (1 + 0.0423 * (t_sfc - t_insitu))
     return pco2_potential
+
+
+def spco2_sensitivity(ds):
+    """Generate sensitivities in spco2 for changes in other variables.
+
+    * Lovenduski, Nicole S., Nicolas Gruber, Scott C. Doney, and Ivan D. Lima.
+        “Enhanced CO2 Outgassing in the Southern Ocean from a Positive Phase of
+        the Southern Annular Mode.” Global Biogeochemical Cycles 21, no. 2
+        (2007). https://doi.org/10/fpv2wt.
+    * Gruber and Sarmiento, 2005
+
+    Args:
+        ds (xr.Dataset): containing cmorized variables:
+                            spco2 [ppm]: pCO2,ocean at ocean surface
+                            talkos[mmol m-3]: Alkalinity at ocean surface
+                            dissicos[mmol m-3]: DIC at ocean surface
+                            tos [C] : temperature at ocean surface
+                            sos [psu] : salinity at ocean surface
+
+    Returns:
+        sensitivity (xr.Dataset):
+
+    """
+    DIC = ds['dissicos']
+    ALK = ds['talkos']
+    SALT = ds['sos']
+    pCO2 = ds['spco2']
+    buffer_factor = dict()
+    buffer_factor['ALK'] = -ALK**2 / ((2 * DIC - ALK) * (ALK - DIC))
```

Comment:
These need to be mean values. So I would go cmorized and say `-ALK.mean('time')`, etc. Correct?

Reply:
I thought the buffer factor needs to be some value around 10